### ISSUE TITLE
testset is used in training function

### ISSUE THREAD

**Issue by Lobotomy92** [kumar-shridhar/PyTorch-BayesianCNN#2]
In line 173 the m is computed as
m = math.ceil(len(testset) / cf.batch_size)
However, it is in the training function, should it be len(trainset)? Because usually the test set should not be seen in the training phase.

**Comment by kumar-shridhar** [comment]
Hi @Lobotomy92 ,

It was a typo in the latest commit. Thanks for raising the issue. Noted and changed.
